doc/databinding/@bindAll.md

```diff
@@ -4,6 +4,103 @@
+@component
+class MyComponent extends Composite {
+
+  @binAll({
+    myText: '#input1.text',
+    myNumber: '#input2.selection'
+  })
+  model: Model;
+
+  // ... constructor that creates input1 and input2 ...
+
+}
+```
+
+In JavaScript the only difference is how - if at all - the decorated property is typed. In JavaScript JSDoc comments may be used, but this is optional. This is true for all examples below, so only the TypeScript variant will be given.
+
+```ts
+/** @type {Model} */
+@binAll({
+  myText: '#input1.text',
+  myNumber: '#input2.selection'
+})
+model;
+```
+
+The bindings are established after `_append` is called the first time on the component. At that time there needs to be exactly one descendant widget with the given id for each binding, and they have to have a property of the same type as the source property. It is okay for the component property to be `null` at that time. The object in the `model` property can also be replaced with another one at any time and the bindings will update accordingly.
+
+### Properties eligible for bindings
+
+Both source and target property need to generate change events for the two-way binding to work. The recommended way to implement this is using [`@property`](./@property.md):
+
+```ts
+class Model {
+  @property myText: string;
+  @property myNumber: number;
+}
+```
+
+Note that there is no need to explicitly create an event API, `@bind` can 'talk' directly to `@property`. However, an explicit implementation is also possible:
+
+```ts
+class Model {
+
+  @event onMyTextChanged: ChangeListeners<Model, 'myText'>;
+  private _myText: string;
+
+  set myText(value: string) {
+    if (this._myText !== value) {
+      this._myText = value;
+      this.onMyTextChanged.trigger({value});
+    }
+  }
+
+  get myText() {
+    return this._myText;
+  }
+
+}
+```
+
+### Edge Cases
+
+The component property (`model` in the above example) may also be set to `null` (or `undefined`) at any time. In that case all the target properties of the child elements will be set back to their initial values. The initial value in this case refers to the value a target property had the moment the target element was attached to the component. The source properties on the former source object will retain their latest value.
+
+When a new two-way binding is established (when `_append` is called or the component property is assigned a new source object) all the target properties will be set to the current value of the their respective source property.
+
+There is one exception to this behavior: If the source property is set to `undefined` (but not `null`) at that moment it will be assigned the current value of the target property. If a source property is set to `undefined` later both properties are set to the initial value of the target property.
+
+If a source property converts or ignores the incoming value of the target property, the target property will follow and change again to contain the new source property value.
```

Comment:
You mean that the value will be converted by the source property's converter and the result will be passed to the target property's converter that will pass it to the native setter (that might change the value again)?

> the target property will follow and change again to contain the new source property value

I found the "will follow and change again ..." part confusing.